packages/api/src/fullApi.ts

```diff
@@ -34,8 +35,8 @@ export class SubsocialApi {
   }
 
   private async findDataArray<S extends CommonStruct, C extends CommonContent> (
-    ids: SubstrateId[],
-    findStructs: (ids: SubstrateId[]) => Promise<S[]>,
+    ids: SupportedSubstrateId[],
+    findStructs: (ids: any[]) => Promise<S[]>,
```

Comment:
Try `SupportedSubstrateId` instead of `any`?
```suggestion
    findStructs: (ids: SupportedSubstrateId[]) => Promise<S[]>,
```